workshop/guide/src/prelimenaries/ebnf.md

```diff
@@ -1 +1,42 @@
-# EBNF
+# Grammars and EBNF
+
+Before diving in EBNF (Extended Backus-Naur Form), we need to have a simple understanding on grammars.
+
+## Grammars
+A grammar is a set of rules a language adheres to. Every language has a grammar, natural languages (e.g. Dutch or
```

Comment:
We defined a languages to be any collection of words. With this interpretation it is not strictly true that "Every language has a grammar", if grammars need to be finite.